styx-client/src/main/java/com/spotify/styx/client/StyxOkHttpClient.java

```diff
@@ -395,13 +410,16 @@ static StyxClient create(String apiHost, FutureOkHttpClient client, GoogleIdToke
   }
 
   private CompletionStage<Response> execute(Request request) {
-    final Optional<String> authToken;
-    try {
-      authToken = auth.getToken(apiHost.toString());
-    } catch (IOException | GeneralSecurityException e) {
-      // Credential probably invalid, configured wrongly or the token request failed.
-      throw new ClientErrorException("Authentication failure: " + e.getMessage(), e);
-    }
+    var authToken = Optional
+        .ofNullable(AUTHORIZATION_KEY.get())
```

Comment:
Wow this is kind of magic. I had go and read about GRPC to understand why this makes sense